app/assets/stylesheets/components/about.scss

```diff
@@ -0,0 +1,198 @@
+#about {
+
+    .row {
+        padding: 40px;
+        color: $text-primary;
+    }
+    
+    .intro {
+        text-align: center;
+        width: 70%;
+        margin: auto;
+        h1 {
+            color: $primary;
+            width: 100%;
+        }
+    }
+    
+    .mission {
+        .row {
+            width: 70%;
+            margin: auto;
+        }
+        h2 {
+            text-align: center;
```

Comment:
Use the `text-center` bootstrap class in the `h2` element instead of this style definition